src/View.php

```diff
@@ -87,17 +93,17 @@ class View implements DynamicContentAwareInterface
     private array $dynamicPlaceholders = [];
 
     /**
-     * @var string $language
+     * @var string
      */
     private string $language = 'en';
 
     /**
-     * @var LoggerInterface $logger
+     * @var LoggerInterface
      */
     private LoggerInterface $logger;
 
     /**
-     * @var string $sourceLanguage
+     * @var string
```

Comment:
all comments that do not carry a meaningful description or payload should be removed

Reply:
Thank. Fixed.